getting-started/android-frontend-with-appengine-backend/README.md

```diff
@@ -2,11 +2,12 @@ Serverless Kotlin: Getting Started App
 ===
 
 This is a sample application to demonstrate a Kotlin backend which communicates with a mobile
-frontend. The backend emojifies people faces in an input image by swapping faces detected on the image with emojis corresponding to predicted emotions. The sample is called **Emojify**.
+frontend. The backend emojifies people faces in an input image by overlaying on detected faces, emojis corresponding to predicted emotions. The sample is called **Emojify**.
```

Comment:
This will be more clearly written as:

```
The backend receives an input image and overlays an emoji corresponding to the emotion on each of the detected faces.
```